setup.cfg

```diff
@@ -72,16 +72,8 @@ source = vorta
 omit = tests/*
 relative_files = true
 
-[flake8]
-ignore =
-max-line-length = 120
-extend-ignore = E203,E121,E123,E126,E226,E24,E704,W503,W504
-exclude =
-    build,dist,.git,.idea,.cache,.tox,.eggs,
-    ./src/vorta/__init__.py,.direnv,env
-
 [tox:tox]
-envlist = py36,py37,py38,flake8
+envlist = py36,py37,py38,ruff
```

Comment:
Does ruff need it's own Python env? Wasn't it a Rust binary?

Reply:
As can be seen from those old Python versions, the tox config isn't maintained any more. So I'd just run ruff in pre-commit hooks which are the main mechanism for this right now.

Reply:
If the tox is unmaintained and no longer used I guess we should remove in a seperate PR?